### PR TITLE
Calendar API loading bug

### PR DESCRIPTION
**Pull request by navro**
Fixes a bug in the calendar where the API was called with null / NaN dates. Also lints and cleans up the file a little :)

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -13,7 +13,6 @@
     "monthNames": true,
     "API": true,
     "BASE_URL": true,
-    "event": true,
```
Comment (navro) [inline .eslintrc.json]: @olofenglund96 Do you remember why you added this here?

File: www/js/calendar.js [hunk 2]
```diff
@@ -1,19 +1,18 @@
 // Init calendar if not already inited
-$$(document).on('page:init', '.page[data-name="calendar"]', function (e) {
-  var page = $('.page.calendar-page');
+$$(document).on('page:init', '.page[data-name="calendar"]', function () {
+  const page = $('.page.calendar-page');
```
Comment (navro) [inline www/js/calendar.js]: @fsek/devs Only very old android devices don't support const/let now, and we can't run the app on those devices anyway. So there's no reason to keep using var anymore :)

Comment (navro) [reply www/js/calendar.js]: We should add that to eslint

File: config.xml [hunk 3]
```diff
@@ -60,6 +60,8 @@
     <preference name="AutoHideSplashScreen" value="false" />
     <preference name="phonegap-version" value="cli-7.1.0" />
     <preference name="HeaderColor" value="#eb7125" />
+    <engine name="android" spec="^6.4.0" />
+    <engine name="ios" spec="^4.5.4" />
```
Comment (FredrikLastow) [inline config.xml]: Amazing change 👍

Comment (navro) [reply config.xml]: I know, it's the best thing with the entire PR :D

**Pull request merged**